sdk/core/azure-core-test-junit-extensions/README.md

```diff
@@ -0,0 +1,52 @@
+# Azure Core Test JUnit extensions shared library for Java
+
+Library containing JUnit extensions used to test Azure SDK client libraries.
+
+## Getting started
+
+To use this package, add the following to your _pom.xml_.
+
+[//]: # ({x-version-update-start;com.azure:azure-core-test-junit-extensions;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-core-test-junit-extensions</artifactId>
+  <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+## Key concepts
+
+
+## Examples
+
```

Comment:
Will this be added in a follow-up PR?

Reply:
Yeah, I left TODO in sdk/core/azure-core-test-junit-extensions/src/samples/java/com/azure/core/test/junit/extensions/ReadmeSamples.java